plugins/modules/azure_rm_backuppolicy.py

```diff
@@ -166,6 +188,48 @@
     type: str
     returned: always
     sample: Microsoft.RecoveryServices/vaults/backupPolicies
+properties:
+    description:
+        - Attributes of the backup policy.
+    type: complex
```

Comment:
```suggestion
    type: dict
```

Reply:
addressed via rebase on top of your commits in #1890